src/marketing/google-universal-analytics.md

```diff
@@ -32,4 +32,7 @@ Visit the Google website, and sign up for a [Google Universal Analytics][1] acco
 
 1. When complete, click <span class="btn">Save Config</span>.
 
+{:.bs-callout-warning}
+If a [Cookie Restriction Mode]({% link stores/compliance-cookie-restriction-mode.md %}) is enabled on a website and a visitor is not accepted cookies then Google Analytics data will be not sent.
```

Comment:
```suggestion
Enabling the [Cookie Restriction Mode]({% link stores/compliance-cookie-restriction-mode.md %}) prevents Google Analytics from collecting data on your website.
```

Reply:
I've impelemented a suggestion, but to double check here.

Magento_GoogleAnalytics module will NOT SEND the analytics data to Google in case when:
- Cookie Restriction Mode = ENABLED
- Customer/guest IS NOT allowed a website cookies

Magento_GoogleAnalytics module will SEND the analytics data to Google in case when:
- Cookie Restriction Mode = ENABLED
- Customer/guest has ALLOWED a website cookies

@jeff-matthews , could you pease verify that. 

Thank you!

Reply:
I see what you mean now @atwixfirster. My suggested edit changed the meaning and it should include a statement about the user accepting cookies. For example:

> If you enable the [Cookie Restriction Mode]({% link stores/compliance-cookie-restriction-mode.md %}), Google Analytics will not collect data about visitors unless they have accepted cookies.

Reply:
Awesome. **done**

@jeff-matthews , please review! Thank you and have a wonderful day!